spring-kafka/src/main/java/org/springframework/kafka/retrytopic/RetryTopicConfigurationBuilder.java

```diff
@@ -71,6 +71,7 @@ public class RetryTopicConfigurationBuilder {
 	@Nullable
 	private BinaryExceptionClassifierBuilder classifierBuilder;
 
+	@SuppressWarnings("deprecation")
 	private FixedDelayStrategy fixedDelayStrategy = FixedDelayStrategy.MULTIPLE_TOPICS;
```

Comment:
This needs to be fully qualified (and remove the `import`). Otherwise the compiler will emit a deprecation warning for the import.

Reply:
Well, there is no import in the top of the class. It is in the same package.

Reply:
Doh - sorry.